### ISSUE TITLE
Builder should handle larger logs 

### ISSUE THREAD

**Issue by vishal-biyani** [fission/fission#777]
Builder handles build logs of packages but probably can't handle too long logs right now. When builing a Java project - which has sufficiently long build logs, following error is seen:

```
Error reading cmd output: bufio.Scanner: token too long
2018/07/04 05:01:56 Error building source package: Error reading cmd output: bufio.Scanner: token too long

```

This comes from github.com/fission/fission/builder/builder.go:

```
// Runtime logs
	for scanner.Scan() {
		output := scanner.Text()
		fmt.Println(output)
		buildLogs += fmt.Sprintf("%v\n", output)
	}

	if err := scanner.Err(); err != nil {
		scanErr := errors.New(fmt.Sprintf("Error reading cmd output: %v", err.Error()))
```
The builder should be able to handle larger logs or chunk and stream log data so that larger data can be handled.

**Comment by erwinvaneyk** [comment]
Yeah, I think you cannot use a `bufio.Scanner` for large files/blobs. Probably should be fixed by just using a Reader. (see https://golang.org/pkg/bufio/#Scanner)

**Comment by soamvasani** [comment]
We'll also run into the CRD size limit when we try to put logs into the status.  We might need to extend the package status type to be able to use a URL for the logs, and put the logs into the storage svc.

**Comment by sanketsudake** [comment]
This has been inactive for 1722 days. We're scrubbing the backlog and closing stale items to keep it actionable. If this is still relevant, please reopen or comment and we'll pick it back up.
